run.sh

```diff
@@ -2,50 +2,70 @@
 
 : "${GF_PATHS_CONFIG:=/etc/grafana/grafana.ini}"
```

Comment:
I'll put the defaults in the Dockerfile using `ENV`, by doing so one can overwrite the default and it's clear where they are set.